README.md

```diff
@@ -12,6 +12,24 @@ conda activate yeat
 pip install .
 ```
 
+### Bandage
+
+In order to run [`Bandage`](https://github.com/rrwick/Bandage) in YEAT's workflow, users will need to manually download the pre-built binaries files [here](https://rrwick.github.io/Bandage/) or build it from the source code [here](https://github.com/rrwick/Bandage). Instructions for building Bandage from the source code can be found [here](https://github.com/rrwick/Bandage#building-from-source).
+
+If the pre-built binary files are used, users will need to update their environment's `$PATH` to include the path of the directory containing the binary file.
```

Comment:
The `$PATH` needs to be set whether the user uses pre-built binaries or builds the binaries from scratch, right?

Reply:
> If the pre-built binary files are used, users will need to update their environment's `$PATH` to include the path of the directory containing the binary file.

Updated this sentence to...

> Once the binary file has been obtained, users will need to update their environment's `$PATH` to include the path to the directory containing the binary file.

Misunderstood that when building from the source, it creates a binary file, which still requires the user to update their $PATH.